templates/resources/vmaas_load_balancer_profile.md.tmpl

```diff
@@ -0,0 +1,68 @@
+---
+layout: ""
+page_title: "hpegl_vmaas_load_balancer_profile Resource - vmaas-terraform-resources"
+subcategory: {{ $arr := split .Name "_" }}"{{ index $arr 1 }}"
+description: |-
+  {{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# Resource hpegl_vmaas_load_balancer_profile
+
+-> Compatible version >= 5.4.6
+
+{{ .Description | trimspace }}
+`hpegl_vmaas_load_balancer_profile` resource supports NSX-T Load balancer Profile creation.
+
+For creating an NSX-T Load balancer Profile, use the following examples.
+
+-> NSX-T Profile having HTTP,UDP,TCP,COOKIE,SOURCEIP,GENERIC,SERVER,CLIENT  considered as different Profile types. You can create either any of them HTTP,HTTPS,ICMP,Passive,TCP,UDP at a given time
+
+## Example usage for creating NSX-T Load balancer Profile for HTTP with all possible attributes
+
+-> ProfileType value is `application-profile` 
```

Comment:
remove these notes "-> ProfileType value is" from everywhere

Reply:
removed